backend/src/models/user.model.ts

```diff
@@ -0,0 +1,17 @@
+import { Schema, model } from 'mongoose';
+
+interface UserModel {
+    first_name: string,
+    last_name: string,
+    user_name: string,
+    persons: [Schema.Types.ObjectId]
+}
+
+const schema = new Schema<UserModel>({
+    first_name: { type: String, required: true },
+    last_name: { type: String, required: true },
+    user_name: { type: String, required: true },
```

Comment:
would this be manually set by the user upon signing up? if we are going to use firebase auth is this possible?

Reply:
this is asking about `user_name` btw

Reply:
Not entirely sure myself, I was just implementing the design in the ticket. What do you think we should do here?

Reply:
I feel like it could be fine to leave it in for now while we figure out how account creation/authentication is going to work, but perhaps set required to false for now?

Reply:
Yeah that's a good compromise, I'll do that for now.